sql/core/src/test/resources/sql-tests/inputs/collations.sql

```diff
@@ -77,3 +77,18 @@ select array_distinct(array('aaa' collate utf8_binary_lcase, 'AAA' collate utf8_
 select array_union(array('aaa' collate utf8_binary_lcase), array('AAA' collate utf8_binary_lcase));
 select array_intersect(array('aaa' collate utf8_binary_lcase), array('AAA' collate utf8_binary_lcase));
 select array_except(array('aaa' collate utf8_binary_lcase), array('AAA' collate utf8_binary_lcase));
+
+-- array joins & aggregates
+create table l(a array<string collate utf8_binary_lcase>) using parquet;
```

Comment:
Seems like golden file examples are already covered by `CollationSuite`? We don't need both.